src/Concerns/Comparable.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Spatie\Enum\Concerns;
+
+trait Comparable
+{
+    /**
```

Comment:
You can remove redundant docblocks

Reply:
Oh yeah, this one was here before I've applied the spreaded type-hint 😂

Reply:
https://github.com/spatie/enum/pull/96/commits/b9250b5501a0af206869b2ad47fde22d3d4bdcb2